libs/s25main/Settings.h

```diff
@@ -63,6 +63,7 @@ class Settings : public Singleton<Settings, SingletonPolicies::WithLongevity>
         bool fullscreen;
         bool vbo;
         bool shared_textures;
+        unsigned guiScale;
```

Comment:
```suggestion
        unsigned guiScale; /// in %, 0="automatic"
```

Reply:
FYI, `///` attributes the documentation comment to the following item. Use `///<` for trailing comments (see https://www.doxygen.nl/manual/docblocks.html#memberdoc), which reminds me that I wanted to update the affected comments in this PR.

Reply:
yep, `///< xxx` is the right way